### ISSUE TITLE
Akten in Projekten zusammenfassen / gruppieren

### ISSUE THREAD

**Issue by j-dimension** [jlawyerorg/j-lawyer-org#41]
Anwenderwunsch: 
"Können Unterakten/verbundene Akten angelegt werden? (Dies wäre v.a. sinnvoll bei zusammengehörenden Fällen, z.B. Gruppen von Schutzrechten.)
Da v.a. im Patent- und Markenrecht Akten häufig zusammenhängen (z.B. einzelne Länderakten), wäre das Fehlen dieser Funktion leider fast ein „K.O.-Kriterium“ bezüglich der Verwendung der Software"

**Comment by j-dimension** [comment]
Benötigt werden Beziehungen mit selbst konfigurierbaren Namen

**Comment by librelegal** [comment]
Aus meiner Sicht, ist das eine Frage der Beziehung von Akten untereinander, die an der korrekten Adressverwaltung hängt. Ich könnte mir vorstellen. dass das mit dem issue der "Erfassung von Beziehungen zwischen den Kontakten" zusammenhängt, aber nicht identisch sein muss. Es könnte ein Duplikat von sein (
Duplicate of #120?).

**Comment by j-dimension** [comment]
Nein, hier ging es damals um eine Anfrage einer großen Kanzlei, die komplexe Mandate "splitten" und von mehreren Anwälten bearbeiten lassen. Deren individuelle Aufgaben sind dann stark voneinander getrennt und sollen als eigene "Unterakten" geführt werden.

**Comment by Thorsten-usr** [comment]
Ja, ich unterstütze die dringende Bitte des Kollegen aus dem Markenrecht, das brauche ich nämlich auch: Konkrete Einzelakten werden einer "Generalakte" zu- bzw. untergeordnet und sind dort dann verlinkt. Ein Klick auf den Link und man landet in der Einzelakte.

**Comment by j-dimension** [comment]
Umsetzung via [#2688](https://github.com/jlawyerorg/j-lawyer-org/issues/2688)
